scene/gui/tree.h

```diff
@@ -527,6 +530,9 @@ friend class TreeItem;
 
 	void set_value_evaluator(ValueEvaluator *p_evaluator);
 
+	void set_delayed_text_editor_enabled(bool enabled);
+	bool get_delayed_text_editor_enabled() const;
```

Comment:
well those are shorter and basically used only in the engine so will change to the proposed ones